tests/lambda_functions/va_profile/test_va_profile_integration.py

```diff
@@ -306,9 +306,6 @@ def test_va_profile_opt_in_out_lambda_handler_malformed_json(jwt_encoded, event_
     assert response['statusCode'] == 400, 'The request body must be a JSON object convertible to a Python dictionary.'
 
 
-# TODO - The next 3 test functions are highly repetitive.  Is there a way to parametrize them?
```

Comment:
Did we try to parameterize these tests? It seems like a reasonable thing to do. Just wanted to ask the question before we forget this comment existed.

Reply:
I looked over them and decided they're just different enough that parametrization would be hackey.

Reply:
This is exactly why I dislike these TODO comments.  Dave if you end up not parameterizing these in this PR, can you just remove the TODO? Seems unlikely we will ever get to it if not handled now.

Reply:
Okay, just wanted to double check. It wouldn't be worth making them overly complicated just to parameterize them.